Support CDS/MRT format writing for `Time` columns [GSoC21b] (#12027)
astropy/astropy: docs/io/ascii/write.rst

  >>> table['Flux'] = MaskedColumn(table['Flux'], mask=[True, False])
  >>> table['magnitude'] = [u.Magnitude(25), u.Magnitude(-9)]

> For columns with `~astropy.time.Time` values and for columns that are thus, the values are

Reviewer: I don't understand this sentence. Do you mean "columns that contain Time objects are are a time mix-in column?
How about
```suggestion
For columns holding `~astropy.time.Time` values, the values are
```
which can be read as "one (= the column is a Time object) or more (the column is a list of Time objects)".

Author: Ya, that's what I meant there. Your suggestion is far more spot on.
